### DISCUSSION TITLE
multiome analysis

### DISCUSSION THREAD

**Discussion by tanasa** [GreenleafLab/ArchR#1438] [General]
Hi Ryan, 

For Multiome analysis, shall we use the current tutorial that presents the data integration in the chapter : 

https://www.archrproject.com/bookdown/defining-cluster-identity-with-scrna-seq.html

or shall we follow a relatively old script that is described in : 

https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html. 

Thank you !

**Comment by rcorces** [comment]
Just to clarify - Multiome is very different from integration.
Multiome is two datasets acquired simultaneously from the same cell. By definition, these dont require "integration" because you know which cell they came from.
So this is the multiome tutorial - https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html
The other link you supplied is for when you have ATAC and RNA data acquired separately and you wish to combine/integrate them

**Comment by tanasa** [comment]
Thank you Ryan. It is always very impressive, as your answers are so prompt.

If I am add a novice question please : in the absence of scRNAseq data, could we perform all the analysis that are described in the ArchR manual only based on scATAC-seq data ?

**Reply by rcorces** [reply]
Any analysis based on RNA data is, of course, not possible without RNA data. Many manual sections utilize RNA data.

**Comment by tanasa** [comment]
Thank you Ryan ! We may have a time course scATAC-seq datasets. Which analytical method shall we use ?

**Comment by millerkaplanlab** [comment]
Is there another tutorial for more detailed downstream analysis for mutliome datasets. I've processed my data through the pipeline; however, I am looking to identify various gene markers in my dataset....

**Reply by rcorces** [reply]
I'm close to pushing a new release with updated documentation. You can see that in its current state here:
https://github.com/GreenleafLab/ArchR_Website/blob/ui_updates/bookdown/21_MultiomicDataAnalysis.Rmd
But generally speaking, the workflows in ArchR arent unique for multiomic data. you just need to extrapolate from the manual.

**Reply by A-legac45** [reply]
Hello, 

I have a 3 multiomic dataset  which I intergrate according to thier RNA assay by using harmony and used seurat for their pretreatement (filtered mt and found good clustering with my candidate genes).

 Now I would like to isolate in my fragments the same cells and this is working but them I can not add to my archrproject the addGeneExpressionMatrix.

How can I manage this problem?

> projMulti2 <- addGeneExpressionMatrix(input = projMulti2, seRNA = seRNA, strictMatch = TRUE, force = TRUE)

If there is an issue, please report to github with logFile!
Overlap w/ scATAC = 1
Overlap Per Sample w/ scATAC : chimera1=215,chimera2=182,first_RNA=482
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function 'NSBS' for signature '"list"'

thanks

**Reply by rcorces** [reply]
Your `seRNA` object must not have the expected format. I would recommend looking at the tutorial and forcing your seRNA input to match exactly the expected format.

**Reply by A-legac45** [reply]
my serna is built like this 

seRNA2<-SummarizedExperiment(assays=list(counts=seurat.object)) which three different origin.idents

What is the correct format i do not found it?
thanks for your answer

**Reply by brian-1111** [reply]
Hi @rcorces 
Thank you for sharing this tutorial. 
I have a question about the `bias`  argument from `getMarkerFeatures` function. I understand that the character inputs for `bias` account for the potential bias variables. However, I notice that in your script you only uses `log10(Gex_nUMI) `for RNA. Is there a reason for not include the `log10(Gex_nGenes)` in `bias`?

> I'm close to pushing a new release with updated documentation. You can see that in its current state here: https://github.com/GreenleafLab/ArchR_Website/blob/ui_updates/bookdown/21_MultiomicDataAnalysis.Rmd But generally speaking, the workflows in ArchR arent unique for multiomic data. you just need to extrapolate from the manual.

**Reply by rcorces** [reply]
no real reason. they are quite correlated though

**Reply by brian-1111** [reply]
> no real reason. they are quite correlated though
@rcorces 
So in other words using `bias = c("TSSEnrichment", "log10(nFrags)", "log10(Gex_nUMI)", "log10(Gex_nGenes)")` in `getMarkerFeatures` is not necessary to produce a more accurate result is that correct?

**Reply by brian-1111** [reply]
> my serna is built like this
> 
> seRNA2<-SummarizedExperiment(assays=list(counts=seurat.object)) which three different origin.idents
> 
> What is the correct format i do not found it? thanks for your answer

You should follow the tutorial @rcorces provided here https://github.com/GreenleafLab/ArchR_Website/blob/ui_updates/bookdown/21_MultiomicDataAnalysis.Rmd.

`import10xFeatureMatrix` will input your RNA data as `RangedSummarizedExperiment` that you can used to merge with your `ArchR` project

```
seRNA <- import10xFeatureMatrix(
  input = rnaFiles,
  names = names(rnaFiles),
  strictMatch = TRUE
)
```

**Reply by rcorces** [reply]
> So in other words using bias = c("TSSEnrichment", "log10(nFrags)", "log10(Gex_nUMI)", "log10(Gex_nGenes)") in getMarkerFeatures is not necessary to produce a more accurate result is that correct?

I cant be sure as I havent tested this

**Reply by brian-1111** [reply]
> > So in other words using bias = c("TSSEnrichment", "log10(nFrags)", "log10(Gex_nUMI)", "log10(Gex_nGenes)") in getMarkerFeatures is not necessary to produce a more accurate result is that correct?
> 
> 
> 
> I cant be sure as I havent tested this

@rcorces thnaks for your reply. But is there a way I can verify it with my own results?
I'm actually quite new to mitiomics analysis. But based on my knowledge I think using `nGenes` instead of` nUMI` for bias input could potentially lead to a more accurate result since `nGenes` could represent the actual uniquely identified genes. I am curious that why do you consider the UMI to be the only input for adjusting the via of the dataset? Is it because the tutorial is only for the demonstration only?

**Comment by ArutStory** [comment]
Thank you, you really helped!
